src/Sylius/Bundle/ApiBundle/Command/Account/VerifyShopUser.php

```diff
@@ -24,31 +24,24 @@
 class VerifyShopUser implements ChannelCodeAwareInterface, LocaleCodeAwareInterface
 {
     public function __construct(
-        public string $token,
-        private ?string $localeCode = null,
-        private ?string $channelCode = null,
+        protected string $token,
+        protected string $channelCode,
```

Comment:
Why `$channelCode` and `$localeCode` are not nullable in this class, but no others?